packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx

```diff
@@ -126,6 +126,7 @@ export const CallArrangement = (props: CallArrangementProps): JSX.Element => {
           callAdapter={adapter}
           onClose={closePane}
           onFetchAvatarPersonaData={props.onFetchAvatarPersonaData}
+          onFetchParticipantMenuItems={props.callControlProps?.onFetchParticipantMenuItems}
```

Comment:
Here be bugs!

Reply:
squash 'em

Reply:
Great fix. We weren't using our custom menu which is why the snapshots have changed (not just moved from live to hermetic)